### DISCUSSION TITLE
slash in cog

### DISCUSSION THREAD

**Discussion by chungyunglo** [Pycord-Development/pycord#1078] [Q&A] [answered]
is that when main.py have slash_command, then the cog.py's slash_command is no active？
what's wrong with the slash？
and, what is all of the slash's kwargs？

**Comment by MarkoKupresanin** [comment]
You have to load your cogs outside the ``on_ready`` function. I would suggest placing the code to load your cogs right before the on_ready function. There might be another way to get around this issue but I don't know of it.

**Comment by Dorukyum** [comment, marked as answer]
You can load the cogs anytime before on_connect is ran.
